### PR TITLE
Tim Gregg Hippy HW

### PR DESCRIPTION
**Pull request by tomiteo**

### CODE REVIEW

**Comment by tomiteo** [comment]
comfort 4.5/5
completeness 4.5/5

File: css/style.css [hunk 1]
```diff
@@ -1,19 +1,60 @@
...
+}
 
-Body Font Size = 14px
-	 Font Family = "Times New Roman", serif
-	 Font Color = black
-	 Background: #F3EBE9
+body{
+	font-size: 14px;
+	font-family: 'Times New Roman', serif;
+	font-color: black;
```
Comment (jshawl) [inline css/style.css:12]: instead: `color:`

File: css/style.css [hunk 2]
```diff
@@ -1,19 +1,60 @@
...
+.page{
+	background-color: white;
+}
+.4-photos{
+	width: 100%
+}
+
+h1 {
+	font-size: 50px;
+  font-family: 'Arial', 'Helvetica', sans-serif;
```
Comment (jshawl) [inline css/style.css:38]: Be consistent with code indentation

File: css/style.css [hunk 3]
```diff
@@ -1,19 +1,60 @@
...
+  Text-transform: uppercase;
```
Comment (jshawl) [inline css/style.css:39]: use lowercase property names `text-transform`

**Comment by jshawl** [comment]
:+1: `

**Pull request closed without merging**